easybuild/easyconfigs/u/Umpire/Umpire-2025.12.0-gompi-2025b-CUDA-12.9.1.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'Umpire'
+version = '2025.12.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/LLNL/Umpire'
+description = """Umpire is a resource management library that allows the discovery, provision,
+and management of memory on machines with multiple memory devices like NUMA and GPUs."""
+
+toolchain = {'name': 'gompi', 'version': '2025b'}
+
+sources = [{
+    'filename': 'v%(version)s.tar.gz',
+    'git_config': {
+        'url': 'https://github.com/LLNL',
+        'repo_name': 'Umpire',
+        'tag': 'v%(version)s',
+        'recursive': True,
+        'keep_git_dir': True,
+    }
+}]
+checksums = [None]
+
+builddependencies = [
+    ('CMake', '3.31.8'),
+]
+
+dependencies = [
+    ('CUDA', '12.9.1', '', SYSTEM),
+]
+
+configopts = "-DENABLE_FORTRAN=On "
+configopts += "-DENABLE_CUDA=On "
+configopts += "-DENABLE_MPI=On "
+configopts += "-DDENABLE_OPENMP=On "
+configopts += "-DUMPIRE_ENABLE_NUMA=On "
+configopts += "-DUMPIRE_ENABLE_FILE_RESOURCE=On "
```

```suggestion
configopts += "-DUMPIRE_ENABLE_FILE_RESOURCE=On "

configopts += "-DBLT_SOURCE_DIR=${EBROOTLLNLMINBLT} "
configopts += "-Dcamp_DIR=${EBROOTCAMP} "
configopts += "-Dfmt_DIR=${EBROOTFMT} "
```

We can then point Umpire to the correct directories